features/admin.applications.v1/components/settings/general-application-settings.tsx

```diff
@@ -226,6 +229,60 @@ export const GeneralApplicationSettings: FunctionComponent<GeneralApplicationSet
                     message: t("applications:notifications.updateApplication.genericError" +
                         ".message")
                 }));
+            });
+    };
+
+    /**
+     * Handles the toggle change to show confirmation modal.
+     *
+     * @param event - Form event.
+     * @param data - Checkbox data.
+     */
+    const handleAppEnableDisableToggleChange = (event: FormEvent<HTMLInputElement>, data: CheckboxProps): void => {
+        setEnableStatus(data.checked);
+        setShowDisableConfirmationModal(true);
+    };
+
+
+    /**
+     * Disables an application.
+     */
+    const handleApplicationDisable = () => {
+        setIsDisableInProgress(true);
+        const state : string = enableStatus ? "enabled" : "disabled";
```

Comment:
Use `i18n` keys for these and IMO, better to move these inline.
else use some meaningful `var` names.